### ISSUE TITLE
Usability issue when moving mail via drag and drop

### ISSUE THREAD

**Issue by ghost** [nextcloud/mail#5367]
#### Summary

Currently when dragging a mail the folder view is somehow reacting. It seems that all folders are expanded. When dropping the file, all folders are minimized. The folders that were visible initially are also minified. When moving the next file, all folder get expanded again but the scrollbar is at the very top. Rather cumbersome when you want to move the item to a folder further down in the list.

### Feature Request

Do not change the folder view neither when dragging nor when dropping mails.

A single folder could be expanded when hovering with a drag item over a folder. This is what Windows Explorer or Outlook 365 do, too.

**Comment by powerpaul17** [comment]
Yes this makes it very cumbersome if one has for example a lot of folders for archiving..
